### DISCUSSION TITLE
OctoText not listed in Plugin Manager?

### DISCUSSION THREAD

**Discussion by robrugge** [berrystephenw/OctoText#44] [Q&A] [answered]
I'm excited to try this out, but since I got the notification that it was available I'm not able to locate OctoText via the Plugin Manager on any of my printers.  Suggestions?
![image](https://user-images.githubusercontent.com/16689614/113487905-8d6ddc00-946f-11eb-98ac-edf86591ab18.png)

Strangely enough, clicking the repository link shows OctoText as the first item on the website...

**Comment by berrystephenw** [comment, marked as answer]
It is likely that you have a Python 2 distribution. I wrote up how to get past this in this post:
https://github.com/berrystephenw/OctoText/discussions/1#discussioncomment-557346
Let me know if that helps!

**Reply by robrugge** [reply]
I unchecked "Only show plugins that are compatible" and now it comes up - so that's likely it.  Thanks!

**Reply by berrystephenw** [reply]
I'm only guessing - but it might be filtering out plugins that won't install because of incompatibilities with your system.
